docs/api.rst

```diff
@@ -103,6 +103,10 @@ ApplicationCommand
 
 .. autoclass:: ApplicationCommand
     :members:
+    
+.. autofunction:: discord.commands.core.application_command
```

Comment:
is this a decorator? you may want the `:decorator:` option from sphinxcontrib-trio